ADOPTERS.md

```diff
@@ -3,7 +3,7 @@
 **📢 If you're using JSON Schema in your organization, please add your company name to this list.**
 **🙏 It really helps the project to gain momentum and credibility. It's a small contribution back to the project with a big impact.**
```

Comment:
```suggestion
**📢 If you're using JSON Schema in your organization, please add your company name to this list.**

**🙏 It really helps the project to gain momentum and credibility. It's a small contribution back to the project with a big impact.**
```

I know this isn't something you changed, but I ideally want the icons on new lines, so I'll sneak this change in with this PR =]